apps/backoffice-v2/src/Router/Router.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FunctionComponent } from 'react';
+import { env } from '../common/env/env';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { RootError } from '../routes/Root/Root.error';
+import { Root } from '../routes/Root/Root.page';
+import { SignIn } from '../routes/SignIn/SignIn.page';
```

Comment:
I think the common convention is to name the root folder `pages` and not `routes`. WDYT?
```suggestion
import { SignIn } from '../pages/SignIn/SignIn.page';
```

Reply:
Remix uses `routes`, NextJS uses `pages` if the app directory is not in use. I found routes clearer but I'm game either way.